src/test/java/com/synacy/lesson01/exercises/ProductTest.java

```diff
@@ -0,0 +1,20 @@
+package com.synacy.lesson01.exercises;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+
+/**
+ * Created by kenichigouang on 5/3/17.
+ */
+public class ProductTest {
+    @Test(expected = IllegalArgumentException.class)
+    public void setUnitPrice_shouldThrowIllegalArgumentExceptionAtUnitPriceNeg1() throws Exception {
+        Product product = mock(Product.class);
```

Comment:
`Product` object should not be mocked because we are testing its feature. Therefore, we want to use the real `Product` object, not the mocked one.